submissions/Vas-P/popup/css/style.css

```diff
@@ -0,0 +1,199 @@
+*{
+	padding: 0;
+	margin: 0;
+}
+
+ul{
+	display: flex;
+	align-items: center;
+}
+
+li{
+	list-style: none;
+	cursor: pointer;
+}
+
+a{
+	display: block;
+	border: 1px solid white;
+	text-decoration: none;
+	color: #989898;
+}
+
+a:focus {
+	border: 1px solid #989898;
+}
+
+nav{
+	display: flex;
+	flex-direction: row-reverse;
+	padding-right: 40px;
+}
+
+.no:hover{
+	text-decoration: underline;
+}
+
+.toggle{
+	position: absolute;
+	cursor: pointer;
+	opacity: 0;
+}
+
+.images:hover{
+	text-decoration: underline;
+}
+
+.nav .bell{
+	margin: 10px;
+}
+
+.bell > a > img{
+	width: 20px;
+	height: 20px;
+}
+
+.popup-button{
+	cursor: pointer;
+}
+
+.popup-button::before{
```

Comment:
Logically this triangle must be pseudoelement of popup body.
In case when you will use this popup in another place you will need to add this triangle to a new place, so better to keep it as part of popup body.